src/docs/product/integrations/amazon-sqs/index.mdx

```diff
@@ -7,10 +7,21 @@ redirect_from:
   - /workflow/global-integrations/integrations/amazon-sqs/
 ---
 
-This is commonly useful in places where you may want to do deeper analysis on exceptions, or empower other teams, such as a Business Intelligence function.
+Amazon SQS is useful for a more in-depth analysis of exceptions, making it quick and easy to pipe exceptions back into your own systems. Or, use it to empower other teams, such as a Business Intelligence function.  
 
-Configuring Data Forwarding is done by visiting **[Project] » Settings » Data Forwarding** and filling in the required information for the given integration.
+This integration needs to be set up in each project for which you wish to use it. It is maintained and supported by the [Sentry community](https://forum.sentry.io/).
 
-Integration with Amazon SQS makes it quick and easy to pipe exceptions back into your own systems.
+## Install and Configure
 
-The payload for Amazon is identical is our standard API event payload, and will evolve over time. For more details on the format of this data, see our [API documentation](/api/events/get-project-event-details/).
+<Alert title="Note" level="info">
+Sentry owner or manager permissions permissions are required to install this integration.
+</Alert>
+
+Navigate to **Settings > [Organization] > Integrations > Integration Details**
```

Comment:
@MimiDumpling I think it makes more sense to say Integrations > Amazon SQS

Reply:
Also maybe just Settings > Integrations. Since when you click on settings in the side bar you don't have to select the organization